### PR TITLE
Set correct publication state for prod entities

### PR DESCRIPTION
**Pull request by MKodde**
The detail screen showed illegal entity options for the published entities.

https://www.pivotaltracker.com/story/show/175535045

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ cache:
 
 before_script:
   - phpenv config-add travis.php.ini
+  - composer self-update 1.10.16
```
Comment (pablothedude) [inline .travis.yml:23]: Maybe it's better to use: `composer self-update --1`

File: src/Surfnet/ServiceProviderDashboard/Application/Service/EntityService.php [hunk 2]
```diff
@@ -352,4 +355,15 @@ private function updateEntityStatusWithJiraTicketStatus(ManageEntity $entity, Is
...
         }
     }
+
+    private function updateStatus(ManageEntity $entity)
+    {
+        $excludeFromPush = $entity->getMetaData()->getCoin()->getExcludeFromPush();
+        if ($excludeFromPush === '1') {
+            $entity->updateStatus(Constants::STATE_PUBLICATION_REQUESTED);
+        }
+        if ($excludeFromPush === '0') {
```
Comment (pablothedude) [inline src/Surfnet/ServiceProviderDashboard/Application/Service/EntityService.php:365]: Consider throwing an exception when no valid excludeFromPush value is met e.g. '-1'.

**Review by pablothedude — APPROVED** [review]
Please add a link to the issue in Pivotal if there is one.
the changes themselves are looking good.

File: .travis.yml (hunk 1 above)
Comment (MKodde) [reply .travis.yml:23]: Superb advice!

File: src/Surfnet/ServiceProviderDashboard/Application/Service/EntityService.php (hunk 2 above)
Comment (MKodde) [reply src/Surfnet/ServiceProviderDashboard/Application/Service/EntityService.php:365]: That is certainly possible, but would mess up test entities who might not have the exclude from push coin. So I either had to add catch clauses around the method call or leave as is. I opted for the latter.

**Pull request merged**